src/browser/provider/pool.js

```diff
@@ -0,0 +1,153 @@
+import { assignIn } from 'lodash';
+import Promise from 'pinkie';
+import { PathBrowserProvider, LocalBrowserProvider, RemoteBrowserProvider } from './built-in-providers';
+import SaucelabsBrowserProvider from './saucelabs';
+import PhantomJSProvider from './phantomjs';
+import BrowserProviderCacheItem from './cache-item';
+import BrowserConnection from '../connection';
+import { GeneralError } from '../../errors/runtime';
+import MESSAGE from '../../errors/runtime/message';
+
+
+const BROWSER_PROVIDER_RE = /^([^:]+)(?::(.*))?$/;
+
+const builtInProviders = {
+    'local':     LocalBrowserProvider,
+    'path':      PathBrowserProvider,
+    'remote':    RemoteBrowserProvider,
+    'saucelabs': SaucelabsBrowserProvider,
+    'phantomjs': PhantomJSProvider
+};
```

Comment:
If you move all built-ins in separate files in dir `built-in-providers`, you can `export default` this object from `index.js` in that dir